src/components/ListItem/SettingsItem.js

```diff
@@ -206,11 +166,11 @@ class SettingsListItem extends React.Component<Props> {
     }
 
     return (
-      <ButtonWrapper onPress={onPress} theme={theme}>
+      <NativeTouchable onPress={onPress}>
         {this.renderContent(processedValue)}
-      </ButtonWrapper>
+      </NativeTouchable>
     );
   }
 }
 
-export default withTheme(SettingsListItem);
```

Comment:
Not sure, but wouldn't this affect `themedColors` variables here?

Reply:
Nope :)
`withTheme` wrapper allows to access theme as a prop inside the wrapped component.
Since `ButtonWrapper` is changed into reusable component, prop `theme` is no longer needed, thus withTheme wrapper is not needed here as well.

`themedColors` that are reachable in styled components are provided by `ThemeProvider` (by Styled Components) that wraps app's root at App.js and stay the same throughout all components (except if altered theme is passed into component as a prop `theme`).